src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -795,8 +795,6 @@ private function checkDefinition($id, array $definition, $file)
                 if ($throw) {
                     throw new InvalidArgumentException(sprintf('The configuration key "%s" is unsupported for definition "%s" in "%s". Allowed configuration keys are "%s".', $key, $id, $file, implode('", "', $keywords)));
                 }
-
-                @trigger_error(sprintf('The configuration key "%s" is unsupported for service definition "%s" in "%s". Allowed configuration keys are "%s". The YamlFileLoader object will raise an exception instead in Symfony 4.0 when detecting an unsupported service configuration key.', $key, $id, $file, implode('", "', $keywords)), E_USER_DEPRECATED);
```

Comment:
The deprecation message mentions an exception that will be risen in 4.0. That's not yet happening with this change, is it?